webpack/scenes/SmartProxy/ExpandedSmartProxyRepositories.js

```diff
@@ -4,17 +4,40 @@ import PropTypes from 'prop-types';
 import { translate as __ } from 'foremanReact/common/I18n';
 import { DataList, DataListItem, DataListItemRow, DataListItemCells, DataListCell } from '@patternfly/react-core';
 import AdditionalCapsuleContent from './AdditionalCapsuleContent';
+import InactiveText from '../ContentViews/components/InactiveText';
+import RepoIcon from '../ContentViews/Details/Repositories/RepoIcon';
+
+const ExpandedSmartProxyRepositories = ({
+  contentCounts, repositories, syncedToCapsule, envId,
+}) => {
+  const filterDataByEnvId = () => {
+    const filteredData = {};
+
+    Object.keys(contentCounts).forEach((key) => {
+      const entry = contentCounts[key];
+      if (entry.metadata.env_id === envId) {
+        filteredData[key] = entry;
+      }
+    });
+
+    return filteredData;
+  };
+  const envContentCounts = filterDataByEnvId();
 
-const ExpandedSmartProxyRepositories = ({ contentCounts, repositories }) => {
   const getRepositoryNameById = id => (repositories.find(repo =>
-    Number(repo.id) === Number(id)) || {}).name;
+    Number(repo.id) === Number(id) || Number(repo.library_id) === Number(id)) || {}).name;
 
-  const dataListCellLists = (repo) => {
+  const dataListCellLists = (repoCounts, repo) => {
     const cellList = [];
     /* eslint-disable max-len */
-    cellList.push(<DataListCell key={`${repo.id}-name`}><span>{getRepositoryNameById(repo)}</span></DataListCell>);
-    cellList.push(<DataListCell key={`${repo.id}-rpm`}><span>{contentCounts[repo].rpm ? `${contentCounts[repo].rpm} Packages` : 'N/A'}</span></DataListCell>);
-    cellList.push(<DataListCell key={`${repo.id}-count`}><AdditionalCapsuleContent counts={contentCounts[repo]} /></DataListCell>);
+    if (syncedToCapsule) {
+      cellList.push(<DataListCell key={`${repo.id}-name`}><span><a href={`/products/${envContentCounts[repo].metadata.product_id}/repositories/${envContentCounts[repo].metadata.library_instance_id}/`}>{getRepositoryNameById(envContentCounts[repo].metadata.library_instance_id)}</a></span></DataListCell>);
+      cellList.push(<DataListCell key={`${repo.id}-type`}><RepoIcon type={envContentCounts[repo].metadata.content_type} /></DataListCell>);
+      cellList.push(<DataListCell key={`${repo.id}-rpm`}><span>{envContentCounts[repo].counts.rpm ? `${envContentCounts[repo].counts.rpm} Packages` : 'N/A'}</span></DataListCell>);
+      cellList.push(<DataListCell key={`${repo.id}-count`}><AdditionalCapsuleContent counts={envContentCounts[repo].counts} /></DataListCell>);
+    } else {
+      cellList.push(<DataListCell key={`${repo.id}-not-synced`}><InactiveText text={__('Content view is not synced to capsule')} /></DataListCell>);
```

Comment:
![image](https://github.com/Katello/katello/assets/14796566/0e576ef6-9fb1-452c-8fd6-174b05065104)

What do you think about showing the repository name and moving the "not synced" text to the count field? That would match up with how I'm doing it in Hammer.